### ISSUE TITLE
Add collapsible block to formattable text (e.g.: pages)

### ISSUE THREAD

**Issue by bane73** [encounterplus/encounterplus#339]
A DM actively-running a campaign needs the key-details of a location/encounter presented as clear and succinctly as possible -- in order to allow them to context switch away from the text and back again quickly and easily. But the DM often needs to know other details too, details which would be nice to have readily-available but which clutter/obscure the most-important details when the two types of information are simply intermixed.

In order to provide this type of support, a collapsible block of text should be able to be added in everywhere that formattable-text is offered. You see a similar feature in DDBeyond, called "spoiler" -- but this feature-suggestion is NOT about hiding from the players; rather, it is intended to hide it from the DM so that it stays out of the way (but close at hand!) until the DM decides they need to consult it.

As a DM/content-creator is creating/adapting their module/campaign, they would have a better result if they could provide key-details and encapsulate less-important details into a collapsible-block *INLINE* with the rest of their content. Just take a look at how DDB implements their "spoiler"-feature but instead of calling it "spoiler" just relabel it to "additional details" or similar.

**Comment by Rycochet** [comment]
In HTML context support for `<details>` + `<summary>` covers this - not sure if it's in there though -


<details>
  <summary>Click to expand!</summary>

  Expanded content to display...
</details>

```html
<details>
  <summary>Click to expand!</summary>

  Expanded content to display...
</details>
```

**Comment by bane73** [comment]
Thank you @Rycochet , this is awesome! Works precisely how I imagined. 
YAY! I really love this platform, thank you to everyone supporting it.

Can we convert this feature request, then, to just adding an HTML-tips/tricks once you go into HTML-mode? Kind of like github does here in this comment box with the "Markdown"-button in the lower-right that opens a page to all of the various options available.
(should this be a new ticket?)

Also I've noticed a small bug when I add the details+summary tags, save the page, and then open back up for edit, it inserts new-lines between the elements... and if I don't manually remove those and repeat the process, it adds additional new-lines for each iteration of save/open-edit... ultimately looking like this:

<p><details><summary>Click to expand!</summary>
    <h4>Details</h4>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
    <p>Expanded details to display here...</p>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
  </details></p>

**Comment by Splinter714** [comment]
@bane73 
Can you confirm this is still an issue? If so, let's get it to it's own separate bug report.
> Also I've noticed a small bug when I add the details+summary tags, save the page, and then open back up for edit, it inserts new-lines between the elements... and if I don't manually remove those and repeat the process, it adds additional new-lines for each iteration of save/open-edit... ultimately looking like this:

**Comment by Splinter714** [comment]
initial issue solved, added remaining request to new issue #480

**Comment by bane73** [comment]
@Splinter714 
Confirmed this looks fixed! I'm running v4.11.1 (2308). Sweet!  :)

> Can you confirm this is still an issue? If so, let's get it to it's own separate bug report.
